general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+
+
+
+I cannot find the field I want to map my column to?
+---------------------------------------------------
+
+Odoo try to find with some heuristic, based on the first ten lines of the files, the type of field for each columns inside your file. For example if you have a column only containing numbers, only the fields that are of type integer will be displayed for you to choose from. While this behaviour might be good and easy for most cases scenarios, it is also possible that it goes wrong sometimes or that you want to map your column to a field that is not proposed by default.
+
+If that happens, you just have to check the **Show all fields for completion (advanced)** option, you will then be able to choose from the complete list of fields for each columns.
+
+
+
+Where can I change the date import format?
+------------------------------------------
+
+Odoo can automatically detect if a column is a date and it will try to guess the date format from a set of most used date format. While this process can work for a lot of simple date format, some exotic date format will not be recognize and it is also possible to have some confusion (day and month inverted as example) as it is difficult to guess correctly which part is the day and which one is the month in a date like '01-03-2016'.
+
+To view which date format Odoo has found from your file you can check the **Date Format** that is shown when clicking on **Options** under the file selector. If this format is incorrect you can change it to your liking using the *ISO 8601* to define the format.
+
+.. note::
+    If you are importing an excel (.xls, .xlsx) file, you can use date cells to store dates as the display of dates in excel is different from the way it is stored. That way you will be sure that the date format is correct in Odoo whatever your locale date format is.
+
+
+
+Can I import numbers with currency sign (e.g.: $32.00)?
+-------------------------------------------------------
+
+Yes, we fully support numbers with parenthesis to represent negative sign as well as numbers with currency sign attached to them. Odoo also automatically detect which thousand/decimal separator you use (you can change those under **options**). If you use a currency symbol that is not known to Odoo, it might not be recognized as a number though and it will crash.
+
+Examples of supported numbers (using thirty-two thousands as an example):
+
+- 32.000,00
+- 32000,00
+- 32,000.00
+- -32000.00
+- (32000.00)
+- $ 32.000,00
+- (32000.00 €)
+
+Example that will not work:
+
+- ABC 32.000,00
+- $ (32.000,00)
+
+
+
+What can I do when the Import preview table isn't displayed correctly?
+----------------------------------------------------------------------
+
+By default the Import preview is set on commas as field separators and quotation marks as text delimiters. If your csv file does not have these settings, you can modify the File Format Options (displayed under the Browse CSV file bar after you select your file).
```

Comment:
maybe add a screeshot here (usually good idea when we need to describe the position of a button)